tests/e2e/tidbcluster/across-kubernetes.go

```diff
@@ -123,27 +126,30 @@ var _ = ginkgo.Describe("[Across Kubernetes]", func() {
 		}
 	})
 
-	ginkgo.Describe("[Basic]", func() {
-
-		// create three namespace
+	ginkgo.Describe("[Deploy]", func() {
```

Comment:
@KanShiori Will `Basic` to `Deploy` affect the CI pipeline run?

Reply:
we haven't created the CI pipeline now 😅

Reply:
No, there's a tag above, named "Across Kubernetes".